install/kubernetes/helm/istio/charts/mixer/templates/deployment.yaml

```diff
@@ -1,88 +1,99 @@
+{{- define "mixer_containers" }}
+    spec:
+      serviceAccountName: istio-mixer-service-account
+      volumes:
+      - name: istio-certs
+        secret:
+          secretName: istio.istio-mixer-service-account
+          optional: true
+    {{- if $.Values.nodeSelector }}
+      nodeSelector:
+{{ toYaml $.Values.nodeSelector | indent 8 }}
+    {{- end }}
+      containers:
+      - name: mixer
+        image: "{{ $.Values.global.hub }}/mixer:{{ $.Values.global.tag }}"
+        imagePullPolicy: {{ $.Values.imagePullPolicy }}
+        ports:
+        - containerPort: 9091
+        - containerPort: 9093
+        - containerPort: 42422
+        args:
+          - --configStoreURL=k8s://
+          - --configDefaultNamespace={{ $.Release.Namespace }}
+          - --trace_zipkin_url=http://zipkin:9411/api/v1/spans
+          - --logtostderr
```

Comment:
i believe this flag is deprecated.

https://github.com/istio/istio/blob/master/pkg/log/options.go#L210

Reply:
good catch.